### PR TITLE
Mbes/change protobuf contracts to reflect domain models

### PR DESCRIPTION
**Pull request by mbusk88**
Have a look :)

### CODE REVIEW

File: source/PostOffice.Communicator.Tests/DataAvailable/DataAvailableNotificationParserTests.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public void Parse_ValidInput_ReturnsData()
             Assert.Equal(contract.UUID, actual.Uuid.ToString().ToUpper(CultureInfo.InvariantCulture));
             Assert.Equal(contract.MessageType, actual.MessageType.Value);
             Assert.Equal(contract.Origin, actual.Origin.ToString());
-            Assert.Equal(contract.Recipient, actual.Recipient.Value);
+            Assert.Equal(contract.GlobalLocationNumber, actual.Recipient.Value);
```
Comment (defectiveAi) [inline source/PostOffice.Communicator.Tests/DataAvailable/DataAvailableNotificationParserTests.cs:51]: Keep name as Recipient. The type is GLN, but the property is Recipient.

Comment (mbusk88) [reply source/PostOffice.Communicator.Tests/DataAvailable/DataAvailableNotificationParserTests.cs]: Fixed

File: source/PostOffice.Communicator.SimpleInjector/source/PostOffice.Communicator.SimpleInjector/PostOffice.Communicator.SimpleInjector.csproj [hunk 2]
```diff
@@ -75,4 +75,8 @@ limitations under the License.
     <Compile Remove="AssemblyInfo.cs" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (michaelmajgaard) [inline source/PostOffice.Communicator.SimpleInjector/source/PostOffice.Communicator.SimpleInjector/PostOffice.Communicator.SimpleInjector.csproj:78]: Should reference package only. If your remove this ItemGroup, doesn't it still build?

Comment (mbusk88) [reply source/PostOffice.Communicator.SimpleInjector/source/PostOffice.Communicator.SimpleInjector/PostOffice.Communicator.SimpleInjector.csproj]: Fixed

File: source/Energinet.DataHub.PostOffice.Application/Energinet.DataHub.PostOffice.Application.csproj [hunk 3]
```diff
@@ -35,7 +35,7 @@ limitations under the License.
 
     <ItemGroup>
       <PackageReference Include="FluentValidation" Version="9.5.1" />
-      <PackageReference Include="GreenEnergyHub.PostOffice.Communicator" Version="1.0.0-CI-03" />
+      <PackageReference Include="GreenEnergyHub.PostOffice.Communicator.SimpleInjector" Version="1.0.0-CI-04" />
```
Comment (defectiveAi) [inline source/Energinet.DataHub.PostOffice.Application/Energinet.DataHub.PostOffice.Application.csproj:38]: Is this right? I think Communicator is/was the correct package.

File: source/Energinet.DataHub.PostOffice.Common/Energinet.DataHub.PostOffice.Common.csproj [hunk 4]
```diff
@@ -25,7 +25,7 @@ limitations under the License.
 
   <ItemGroup>
     <PackageReference Include="FluentValidation" Version="9.5.1" />
-    <PackageReference Include="GreenEnergyHub.PostOffice.Communicator" Version="1.0.0-CI-03" />
+    <PackageReference Include="GreenEnergyHub.PostOffice.Communicator.SimpleInjector" Version="1.0.0-CI-04" />
```
Comment (defectiveAi) [inline source/Energinet.DataHub.PostOffice.Common/Energinet.DataHub.PostOffice.Common.csproj:28]: Here too.

File: source/Energinet.DataHub.PostOffice.Infrastructure/Energinet.DataHub.PostOffice.Infrastructure.csproj [hunk 5]
```diff
@@ -31,7 +31,7 @@ limitations under the License.
     <ItemGroup>
       <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.3.0" />
       <PackageReference Include="Azure.Storage.Blobs" Version="12.10.0" />
-      <PackageReference Include="GreenEnergyHub.PostOffice.Communicator" Version="1.0.0-CI-03" />
+      <PackageReference Include="GreenEnergyHub.PostOffice.Communicator.SimpleInjector" Version="1.0.0-CI-04" />
```
Comment (defectiveAi) [inline source/Energinet.DataHub.PostOffice.Infrastructure/Energinet.DataHub.PostOffice.Infrastructure.csproj:34]: Here too.

File: source/Energinet.DataHub.PostOffice.Application/Energinet.DataHub.PostOffice.Application.csproj (hunk 3 above)
Comment (mbusk88) [reply source/Energinet.DataHub.PostOffice.Application/Energinet.DataHub.PostOffice.Application.csproj]: Fixed

File: source/Energinet.DataHub.PostOffice.Infrastructure/Energinet.DataHub.PostOffice.Infrastructure.csproj (hunk 5 above)
Comment (mbusk88) [reply source/Energinet.DataHub.PostOffice.Infrastructure/Energinet.DataHub.PostOffice.Infrastructure.csproj]: Fixed

File: source/Energinet.DataHub.PostOffice.Common/Energinet.DataHub.PostOffice.Common.csproj (hunk 4 above)
Comment (mbusk88) [reply source/Energinet.DataHub.PostOffice.Common/Energinet.DataHub.PostOffice.Common.csproj]: Fixed

File: source/PostOffice.Communicator.Tests/Peek/DataBundleRequestSenderTests.cs [hunk 6]
```diff
@@ -53,7 +53,7 @@ public async Task Send_AllIsOk_ReturnsResponse()
             var queue = $"sbq-{domainOrigin}";
             var replyQueue = $"sbq-{domainOrigin}-reply";
             var serviceBusSenderMock = new Mock<ServiceBusSender>();
-            var requestBundleResponse = new RequestBundleResponse { Success = new RequestBundleResponse.Types.FileResource { Uri = "http://localhost", UUID = { new[] { "A8A6EAA8-DAF3-4E82-910F-A30260CEFDC5" } } } };
+            var requestBundleResponse = new DataBundleResponseContract { Success = new DataBundleResponseContract.Types.FileResource { ContentUri = "http://localhost", DataAvailableNotificationIds = { new[] { "A8A6EAA8-DAF3-4E82-910F-A30260CEFDC5" } } } };
```
Comment (FirestarJes) [inline source/PostOffice.Communicator.Tests/Peek/DataBundleRequestSenderTests.cs:56]: Should be split over multiple lines

File: source/PostOffice.Communicator/source/PostOffice.Communicator/Peek/ResponseBundleParser.cs [hunk 7]
```diff
@@ -27,42 +27,42 @@ public byte[] Parse(RequestDataBundleResponseDto requestDataBundleResponseDto)
...
             if (!requestDataBundleResponseDto.IsErrorResponse)
             {
-                contract.Success = new RequestBundleResponse.Types.FileResource { Uri = requestDataBundleResponseDto.ContentUri?.AbsoluteUri };
+                contract.Success = new DataBundleResponseContract.Types.FileResource { ContentUri = requestDataBundleResponseDto.ContentUri?.AbsoluteUri };
                 return contract.ToByteArray();
             }
 
             var contractErrorReason = MapToFailureReason(requestDataBundleResponseDto.ResponseError!.Reason);
-            contract.Failure = new RequestBundleResponse.Types.RequestFailure { Reason = contractErrorReason, FailureDescription = requestDataBundleResponseDto.ResponseError.FailureDescription };
+            contract.Failure = new DataBundleResponseContract.Types.RequestFailure { Reason = contractErrorReason, FailureDescription = requestDataBundleResponseDto.ResponseError.FailureDescription };
```
Comment (FirestarJes) [inline source/PostOffice.Communicator/source/PostOffice.Communicator/Peek/ResponseBundleParser.cs:39]: Should be split over multiple lines to ease readability

File: source/PostOffice.Communicator/source/PostOffice.Communicator/Peek/ResponseBundleParser.cs [hunk 8]
```diff
@@ -27,42 +27,42 @@ public byte[] Parse(RequestDataBundleResponseDto requestDataBundleResponseDto)
...
         {
             try
             {
-                var bundleResponse = RequestBundleResponse.Parser.ParseFrom(dataBundleReplyContract);
-                return bundleResponse!.ReplyCase != RequestBundleResponse.ReplyOneofCase.Success
+                var bundleResponse = DataBundleResponseContract.Parser.ParseFrom(dataBundleReplyContract);
+                return bundleResponse!.ReplyCase != DataBundleResponseContract.ReplyOneofCase.Success
                     ? null
-                    : new RequestDataBundleResponseDto(new Uri(bundleResponse.Success.Uri), bundleResponse.Success.UUID.Select(Guid.Parse).ToList());
+                    : new RequestDataBundleResponseDto(new Uri(bundleResponse.Success.ContentUri), bundleResponse.Success.DataAvailableNotificationIds.Select(Guid.Parse).ToList());
```
Comment (FirestarJes) [inline source/PostOffice.Communicator/source/PostOffice.Communicator/Peek/ResponseBundleParser.cs:50]: Should be split over multiple lines to ease readability

**Review by FirestarJes — CHANGES_REQUESTED** [review]
Just few lines that should be split

**Review by defectiveAi — DISMISSED** [review]

File: source/PostOffice.Communicator.Tests/Peek/DataBundleRequestSenderTests.cs (hunk 6 above)
Comment (mbusk88) [reply source/PostOffice.Communicator.Tests/Peek/DataBundleRequestSenderTests.cs:56]: Done

File: source/PostOffice.Communicator/source/PostOffice.Communicator/Peek/ResponseBundleParser.cs (hunk 7 above)
Comment (mbusk88) [reply source/PostOffice.Communicator/source/PostOffice.Communicator/Peek/ResponseBundleParser.cs:39]: Done

**Review by FirestarJes — APPROVED** [review]

**Review by michaelmajgaard — APPROVED** [review]

**Pull request merged**